docs/reference/search/rrf.asciidoc

```diff
@@ -156,17 +162,26 @@ GET example-index/_search
 ----
 // TEST[skip:example fragment]
 
-In the above example, we execute each of the two sub searches
-independently of each other. First we run the term query for
-`blue shoes sales` using the standard BM25 scoring algorithm. Then
-we run the text expansion query for `What blue shoes are on sale?`
+In the above example, we execute each of the two `standard` retrievers
+independently of each other. First we run the `standard` retriever
+specifying a term query for `blue shoes sales` using the standard BM25
+scoring algorithm. Then we run the `standard` retriever specifying a
+text expansion query for `What blue shoes are on sale?`
 using our <<semantic-search-elser, ELSER>> scoring algorithm.
-RRF allows us to combine the two results sets generated by completely
-independent scoring algorithms with equal weighting. Not only does this
-remove the need to figure out what the appropriate weighting would be
-using linear combination, but RRF is also shown to give improved
+The `rrf` retriever allows us to combine the two top documents sets
+generated by completely independent scoring algorithms with equal weighting.
+Not only does this remove the need to figure out what the appropriate
+weighting is using linear combination, but RRF is also shown to give improved
 relevance over either query individually.
 
+[[rrf-using-sub-searches]]
+==== Reciprocal rank fusion using sub searches
+
+RRF using sub searches is no longer supported. Use the
```

Comment:
I thought we never released sub_searches support to the public. But it seems like we did, yet it was experimental? I wonder if we should even mention it as a migration path.

Reply:
I agree! I actually had removed it, but there are cross references from enterprise-search docs that I wasn't aware of until I got errors here during CI. I figured this allows enterprise-search time to move their documentation to use retrievers instead of sub_searches and we can remove them once they have the opportunity to change their references.

Reply:
I see, can we clarify whether we need to mention the previous way at all?

Reply:
@javanna I chatted with @kderusso about removing these elements from the documentation. She's going to review the docs in this PR, and we will follow up with getting the cross references changed once this PR is merged.